scripts/game.js

```diff
@@ -0,0 +1,116 @@
+function startNewGame() {
+  if (players[0].name === "" || players[1].name === "") {
+    playerNameAlertModal.style.display = "block";
+    return;
+  }
+
+  resetGameStatus();
+
+  activeGameSection.style.display = "block";
+  playerTurnElement.textContent = players[activePlayer].name;
+}
+
+function switchPlayer() {
+  if (activePlayer === 0) {
```

Comment:
Can you explain what the difference between `==` and `===` is?

Reply:
== only compares values and === compares type and value